kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/ManageSyncSchedule.vue

```diff
@@ -0,0 +1,230 @@
+                >
+                  edit
+                </KButton>
+              </td>
+            </tr>
+          </tbody>
+        </template>
+
+      </CoreTable>
+      <KModal
+        v-if="deviceModal"
+        :title="$tr('selectDevices')"
+        size="medium"
+        submitText="CONTINUE"
+        cancelText="CANCEL"
+        @cancel="closeModal"
+        @submit="submitModal"
+      >
+        <KGrid>
+          <KGridItem
+            :layout8="{ span: 4 }"
+            :layout12="{ span: 6 }"
+          >
+            <KButton
+              appearance="basic-link"
+              @click="newAddress"
+            >
+              Add new Address
+            </KButton>
+          </KGridItem>
+        </KGrid>
+
+        <KGrid gutter="48">
+          <KGridItem
+            :layout8="{ span: 4 }"
+            :layout12="{ span: 6 }"
+          >
+
+            <KRadioButton
+              v-model="deviceModal"
+              label="LINUX"
+              value="linux"
+            >
+              <span>localhost:8040</span>
+            </KRadioButton>
+          </KGridItem>
+
+          <KGridItem
+            :layout="{ alignment: 'right' }"
+            :layout8="{ span: 4 }"
+            :layout12="{ span: 6 }"
+          >
+            <KButton
+              appearance="basic-link"
+            >
+              Forget
+            </KButton>
+          </KGridItem>
+        </KGrid>
+      </KModal>
+    </KPageContainer>
+  </ImmersivePage>
+
+</template>
+
+
+<script>
+
+  import ImmersivePage from 'kolibri.coreVue.components.ImmersivePage';
+  import CoreTable from 'kolibri.coreVue.components.CoreTable';
+
+  export default {
+    name: 'ManageSyncSchedule',
+    components: {
+      ImmersivePage,
+      CoreTable,
+    },
+    extends: ImmersivePage,
+
+    data() {
+      return { deviceModal: false };
+    },
+    methods: {
+      closeModal() {
+        this.deviceModal = false;
+      },
+      submitModal() {
+        this.deviceModal = false;
+        this.$router.push('/editsyncschedule');
+      },
+      newAddress() {
+        this.$router.push('/newAddress');
+      },
+      editButton() {
+        this.$router.push('/editdevice');
+      },
+    },
+
+    $trs: {
+      toolbarHeader: {
+        message: 'Sync Schedules',
+        context: "Heading for 'manage sync schedule' page.",
+      },
+      manageSyncTitle: {
+        message: 'Sync Schedules',
+        context: 'Title for the manage sync page',
+      },
+      introduction: {
+        message:
+          'Set a schedule for Kolibri to automatically try syncing with other Kolibri devices that share this facility. Devices must be connected to the same network at the scheduled sync time.',
+        context: 'Introduction on the manage sync schedule',
+      },
+      homeFacility: {
+        message: 'Home Facility Otodi Allan',
+        context: 'My home facility',
+      },
+      addDevice: {
+        message: 'ADD DEVICE',
```

Comment:
You do not need to add the all-capitalized here -- the styling is handled within the button